### DISCUSSION TITLE
Populate initial browser from mpd-0.24 recently added

### DISCUSSION THREAD

**Discussion by ghost** [SoongNoonien/mpdevil#110] [Ideas]
Previous discussions had some consensus that displaying the first artist from the artists list in the initial browser window  could be improved upon.

With the (yet to be released) mpd-0.24 a new feature, "added-since", will be available  which could allow Plattenalbum to populate the initial browser window with recently added albums. This could be a fixed number of days (say 10) or a user configurable option, I have no real preference there.

 I think this would be a good usage scenario but as always I may be missing something.

Thoughts

**Comment by SoongNoonien** [comment]
Yes, this should be possible with 0.24 but I'm still not sure if I like this feature. If I ever implement this it won't be the default. Another small drawback is that when 0.24 is first started it should rebuild the whole database to incorporate the new `added-since` field this could have the effect that possibly all albums appear to be added in the last 10 (or what ever) days. The same would be the case for a freshly configured mpd instance. So in addition to the limit of days we probably need to limit the total amount of recent albums in order to display them efficiently.

**Reply by ghost** [reply]
Good point regarding freshly configured mpd instances, maybe a limit of 30 albums would be sufficient to prevent Plattenalbum bogging down on most systems. If a 30 album limit were applied then I guess that would catch the just upgraded to 0.24 issue as well.

In terms of features there will always be a trade off between personal taste, performance and providing features that end users are expecting from an mpd client. From my perspective Plattenalbum has the potential to be the best mpd client out there! the ui is clean and very slick it really is a pleasure to use, nonetheless I can't get away from feeling it's currently limited to small libraries of 1,000 albums or less.

Just my 2 cents.
